apps/src/aichat/views/UserChatMessageEditor.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '@cdo/apps/templates/Button';
+import styles from './userChatMessageEditor.module.scss';
+
+const UserChatMessageEditor: React.FunctionComponent = () => {
+  return (
+    <div className={styles.UserChatMessageEditor}>
+      <textarea
+        className={styles.textArea}
+        placeholder="Add a chat message..."
+      />
+      <Button
+        key="submit"
+        text="Submit"
+        icon="arrow-up"
+        onClick={() => {
+          console.log('Submit button clicked');
+        }}
+        color="purple"
```

Comment:
Can this be set in CSS?

Reply:
I tried adding to `userChatMessageEditor.module.scss` the following:
```
@import 'color.scss';

.Button {
    color: $purple;
}
```
But the style didn't take. Do you have any suggestions? Thanks!

Reply:
I'm importing `Button` from 'templates/Button.jsx'.

Reply:
Thanks. I did try to add `className={moduleStyles.submitButton}` to `Button` with the following added to the module file:
```
@import 'color.scss';

.submitButton {
    color: $purple;
}
```
But this didn't seem to work either.

Reply:
After discussion offline, will keep `Button` as is.